model_compiler/tests/model_compiler/compilers/test_onnx_model_to_tensorrt_model.py

```diff
@@ -60,7 +72,32 @@ def _make_onnx_model(func, batch_size_1, batch_size_2):
                                session=session)
     )
 
-    return onnx_compiler.compile_source(frozen_graph_model)
+    return tf_onnx_compiler.compile_source(frozen_graph_model)
+
+
+def _make_resnet50_onnx_model_file(file_path):
+    device = torch.device('cpu')
+    dummy_input = Variable(torch.randn(1, 3, 224, 224))
+    dummy_input = dummy_input.to(device)
+    model = torchvision.models.resnet50(pretrained=True)
+    resnet50_onnx_model_path = os.path.join(file_path, 'model', 'resnet50.onnx')
+    dir_name = os.path.dirname(resnet50_onnx_model_path)
+    os.makedirs(dir_name, exist_ok=True)
+    torch.onnx.export(model, dummy_input, resnet50_onnx_model_path, verbose=True, keep_initializers_as_inputs=True)
+    return resnet50_onnx_model_path
```

Comment:
make these code simpler and cleaner, for example: `dummy_input = Variable(torch.randn(1, 3, 224, 224)).to(device)`, ` os.makedirs(os.path.dirname(resnet50_onnx_model_path), exist_ok=True)`, etc.